README.md

```diff
@@ -108,11 +108,11 @@ An example query body might look like:
             ["tags[environment]", "=", "prod"]
         ],
         "from_date": "2011-07-01T19:54:15",
-        "to_date": "2018-07-06T19:54:15"
+        "to_date": "2018-07-06T19:54:15",
         "granularity": 3600,
         "groupby": ["group_id", "time"],
         "having": [],
-        "issues": [],
+        "issues": []
```

Comment:
I don't think `issues` is even a valid field anymore. It would actually be nice to update this with a SnQL query example.

Reply:
re: SnQL. My plan is to actually get rid of most of the README content and direct users here: https://getsentry.github.io/snuba/
once we will have a decent landing page there.
Instead use README only as an introduction like relay does.

Reply:
Also, correct, please remove the issues field entirely.

Reply:
I have removed the issues field.